x-pack/plugins/lens/public/indexpattern_datasource/dimension_panel/dimension_editor.tsx

```diff
@@ -147,7 +147,13 @@ export function DimensionEditor(props: DimensionEditorProps) {
       }
 
       let label: EuiListGroupItemProps['label'] = operationPanels[operationType].displayName;
-      if (isActive) {
+      if (disabledStatus) {
+        label = (
+          <EuiToolTip position="top" content={disabledStatus}>
+            <span>{operationPanels[operationType].displayName}</span>
+          </EuiToolTip>
```

Comment:
Because of the location of this `EuiToolTip` within the markup, the tooltip currently only triggers when hovering the text within the button, but not when hovering any part of the button that doesn't have text (i.e. the right side of the button). 

![image](https://user-images.githubusercontent.com/3884767/100657083-bb091400-331b-11eb-93c2-fe516df7ea82.png)

Ideally, we'd have the `EuiToolTip` component wrap the `EuiListGroupItem`'s `button` element, but I don't think that's possible without additional customizations. Instead, could we add a prop of `display="block"` to `EuiToolTip` in order to make the tooltip anchor stretch the width inside the button?

Reply:
This gives us this ugly effect of tooltip being hooked to empty space:
![image](https://user-images.githubusercontent.com/4283304/100726659-85b90080-33c5-11eb-8aea-f2e055590f44.png)

Let me know what you think.

Reply:
Try applying a prop of `position="left"`. That should remedy your concern about the tooltip being centered on potentially empty space, but still address my concern by allowing most of the hover area to still trigger the tooltip.